### PR TITLE
Small fixes to EN locale.

### PR DESCRIPTION
**Pull request by Septiple**
This one also corrects PyTank labels, so it may be controversial. Let me know if it's undesired.

### CODE REVIEW

File: locale/en/locale.cfg [hunk 1]
```diff
@@ -25,7 +25,8 @@ small-parts-03=Special small parts
 oil-sand=Oil sand
 oil-mk01=Small oil patch
 oil-mk02=Medium oil patch
-oil-mk03=Big oil patch
+oil-mk03=Big oil patchpatch
```
Comment (oorzkws) [inline locale/en/locale.cfg:28]: > patchpatch

**Comment by Septiple** [comment]
Good catch, fixed!

**Comment by oorzkws** [comment]
@kingarthur91 thoughts on the tank re-label?

**Comment by kingarthur91** [comment]
i guess

**Pull request merged**